exercises/timesTable13.js

```diff
@@ -10,7 +10,10 @@
  */
 function timesTable13(n) {
   // This is your job. :)
-
+  for (i = 1; i < n + 1; i++) {
```

Comment:
👍

Reply:
You could also do `for (let i = 1; i <= n; i++) { ... }`. Also, without `let`, `i` here becomes a global variable.